cogs/ahk/ahk.py

```diff
@@ -428,7 +429,18 @@ async def rlx(self, ctx, *, code):
 		'''Compile and run Relax code through CloudAHK. Example: `rlx define i32 Main() {return 20}`'''
 
 		await self.cloudahk_call(ctx, code, 'rlx')
-
+	
+	@commands.command()
+	async def wta(self,ctx):
+		open_category = self.bot.get_channel(OPEN_CATEGORY_ID)
+		channels = list(open_category.text_channels)
+		channel_mentions = []
+		for i, c in enumerate(channels):
+			if c.id == GET_HELP_CHANNEL_ID:
+				continue
+		channel_mentions.append(c.mention)
+		await ctx.send("Please ask in a channel under **{}** category. Use either {} or {}.".format(open_category.name,*channel_mentions))
```

Comment:
It is possible that there could be zero open channels, or just one. In those cases, this message isn't correct, so you'll want to account for `len(channel_mentions)` being less than two.

Reply:
Thank you, fixed in https://github.com/Run1e/AceBot/pull/19.